fastapi-google-stt/app/services/websocket_stt_service.py

```
@@ -5,6 +5,7 @@
 import asyncio
 import threading
 import queue
+import time
```

Ruff (0.8.2)

`time` imported but unused

Remove unused import: `time`

(F401)